e2e/test/ASCforIoTOmsClient.cs

```diff
@@ -0,0 +1,116 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Diagnostics;
+using System.Globalization;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Threading.Tasks;
+using Microsoft.IdentityModel.Clients.ActiveDirectory;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+
+namespace Microsoft.Azure.Devices.E2ETests
+{
+    public class ASCforIoTOmsClient : IDisposable
```

Comment:
Naming... (Please read the dev coding standards. ) What is ASC and Oms?
Please keep in-line with other test file names (`<something>Test.cs`)

Reply:
(After rename, please ensure the .cs file matches the class name.)

Reply:
This class is a helper class used for querying the Log Analytics workspace.
I use this class to validate that the messages have successfully arrived and ingested to the customer Log Analytics workspace. It is like the EventHubTestListener class that used to validate messages have arrived at the customer's event hub.